pipeline/report.py

```diff
@@ -533,6 +533,8 @@ def save_figs(figs, fig_names, dir2save, prefix):
 
 
 def delete_outdated_probe_tracks(project_name='MAP'):
+    dj.config['safemode'] = False
```

Comment:
if this is meant to be permanent, this should probably stay in the top as global - otherwise if you are trying to use temporary should either unset or use the 'with dj.config' syntax